### PR TITLE
Make free shipping from value editable

### LINKED ISSUE #9: Make free shipping from value editable
In the first iteration of this block, a merchant should be able to directly set the free shipping from value via the [Settings Sidebar](https://developer.wordpress.org/block-editor/how-to-guides/block-tutorial/block-controls-toolbar-and-sidebar/#settings-sidebar).

### PR DESCRIPTION
**Pull request by nielslange**
Fixes #9 

## Note

In i1 of this block, the merchant must be able to provide the free shipping value in the editor via the Inspector Controls. This PR adds this functionality.

<table>
<tr>
<td>Before:
<br><br>

#9-before
</td>
<td>After:
<br><br>

#9-after
</td>
</tr>
</table>

## Test instructions

- Check out this PR.
- Create a test page.
- Add the block `Free Shipping Progress Bar` to the test page.
- Click on the block to open the Inspector Controls.
- Adjust the free shipping value to ensure that the progress bar changes.

### CODE REVIEW

File: build/frontend.asset.php [hunk 1]
```diff
@@ -1 +1 @@
-<?php return array('dependencies' => array('wp-element', 'wp-polyfill'), 'version' => '889e873949c9ad8654a574a976c91cc5');
\ No newline at end of file
+<?php return array('dependencies' => array('wp-element', 'wp-polyfill'), 'version' => '77379583903f779f03cdbca8a0c3b6dc');
```
Comment (senadir) [inline build/frontend.asset.php]: I might have forget to mention this, but `/build` should be gitignored.

File: src/block.js [hunk 2]
```diff
@@ -3,7 +3,10 @@ export default function Block( { currentTotal, freeShippingFrom } ) {
 	const divWidth = ( progress > 100 ? 100 : progress ) + '%';
 	const divStyle = { width: divWidth };
 	const remaining = Number( freeShippingFrom - currentTotal ).toFixed( 2 );
-	const message = `Spend $${ remaining } more to get free US shipping.`;
+	const message =
+		remaining > 0
+			? `Spend only $${ remaining } more to get free US shipping.`
+			: `You've qualified for free shipping. Great job!`;
```
Comment (senadir) [inline src/block.js]: Strings should be translatable.

**Review by senadir — APPROVED** [review]
Code wise this is looking good, I left a couple of notes.

File: build/frontend.asset.php (hunk 1 above)
Comment (nielslange) [reply build/frontend.asset.php]: Thanks for pointing this out. I added `build` to the `.gitignore` file and ran the following command to remove the `build` folder from the remote repo:

```
$ git rm -r --cached build
$ git commit -m "Remove build folder from repository"
$ git push origin trunk -f
```

File: src/block.js (hunk 2 above)
Comment (nielslange) [reply src/block.js]: Thanks, Nadir. I just localized these strings.

**Pull request merged**